docs/Libraries.md

```diff
@@ -1,6 +1,9 @@
 ## Working with libraries and modules
 
-Carp comes with a standard library called [Core](../core/) where only certain files are loaded by default (see [Core.carp](../core/Core.carp)). If your `CARP_DIR` environment variable is [set properly](https://github.com/carp-lang/Carp/blob/master/docs/Install.md#setting-the-carp_dir), these libraries will be easily imported using the `load` command. For example, to get access to the Bench module, do the following:
+Carp comes with a standard library called Core, sources are [here](../core/).
+Documentation is available [online](http://carp-lang.github.io/carp-docs/core/core_index.html) as well as [locally](./core/core_index.html).
+Many of these - 44 of 62 at the time of writing - are loaded by default, see [Core.carp](../core/Core.carp) for details. 
```

Comment:
Keeping this number up do date will be obnoxious, so maybe just skip that part?

Reply:
Done.